building-with-ocean/using-ocean-libraries/README.md

```diff
@@ -0,0 +1,14 @@
+# Using Ocean libraries
+
+Ocean Protocol officially supports two client libraries:
+
+* ocean.js
+* ocean.py&#x20;
+
+| ocean.js                                                       | ocean.py                                                       |
```

Comment:
The way they have formatted the table, it's not really clear that this is the heading. Perhaps we could also put it in bold?